File: programs/fizz_buzz.ts
export {};

let n: number = 0;

while (true) {
	if (n % 3 === 0) {
		if (n % 5 === 0) {
			console.log('FizzBuzz');
		} else {
			console.log('Fizz');
		}
	}else if (n % 5 === 0) {
		console.log('Buzz');
	} else {
		console.log(n);
	}

	n += 1;
}

File: programs/fizz_buzz.f90
program fizz_buzz
    implicit none

    integer :: n
    
    n = 1

    do while (.true.)
        if (mod(n, 3) .eq. 0) then
            if (mod(n, 5) .eq. 0) then
                print *, 'FizzBuzz'
            else
                print *, 'Fizz'
            end if
        else if (mod(n, 5) .eq. 0) then
            print *, 'Buzz'
        else
            print *, n
        end if

        n = n + 1

    end do
end program fizz_buzz